tests/test_part_image.py

```diff
@@ -158,7 +158,9 @@ def test_partition_alignment(
             int(bitbake_variables["MENDER_DATA_PART_SIZE_MB"]) * 1024 * 1024
         )
 
-        if "mender-uboot" in bitbake_variables["MENDER_FEATURES"]:
+        if "mender-uboot" == bitbake_variables.get(
+            "MENDER_FEATURES"
+        ) or "mender_uboot" == bitbake_variables.get("DISTRO_FEATURES"):
```

Comment:
Right :+1: 

So this is basically due to updating `mender-convert` to the latest module, correct?